taos/examples/raw-dump.rs

```diff
@@ -9,11 +9,40 @@ use taos::*;
 
 #[derive(Debug, Parser)]
 struct Opts {
+    /// User
+    #[clap(short, long, default_value = "root")]
+    user: String,
+    /// Password
+    #[clap(short = 'P', long, default_value = "taosdata")]
+    pass: String,
     /// The target to connect to.
-    #[clap(
-        default_value = "tmq://localhost:6030/ts5250?group.id=dump&experimental.snapshot.enable=true&auto.offset.reset=earliest"
-    )]
-    tmq: String,
+    #[clap(long, default_value = "localhost")]
+    host: String,
+    #[clap(long, default_value = "6030")]
+    port: u16,
+    /// The topic to consume from.
+    #[clap(short, long, default_value = "restful_pcs")]
+    topic: String,
```

Comment:
It is highly likely that a default topic will not exist. Is it possible to allow users to specify one instead of providing a default value?

Reply:
I mean to remove the default value and let the user specify it, or create this default topic.